building/build-debs/homeworld-admin-tools/src/configuration.py

```diff
@@ -164,6 +164,27 @@ def get_node(self, node_name: str) -> Node:
                 return node
```

Comment:
A few meta notes:
 - Once you're done making changes, please squash your changes into a single commit.
 - Please fix the author so that GitHub recognizes this as your commit.
 - Please sign your commit with your PGP key. (Create a new key if you don't have one yet.)